frontend/javascripts/oxalis/model/actions/skeletontracing_actions.js

```diff
@@ -233,6 +239,12 @@ export const createNodeAction = (
   timestamp,
 });
 
+export const createEdgeAction = (sourceNodeId: number, targetNodeId: number): CreateEdgeAction => ({
+  type: "CREATE_EDGE",
+  sourceNodeId,
+  targetNodeId,
+});
```

Comment:
@daniel-wer added support for cyclic trees (aka graphs ^^) to webKnossos in https://github.com/scalableminds/webknossos/pull/2131, but since then we didn't really encourage creating those. I'm inclined to keep it that way, since there haven't been any requests for this (or were there any?).
So, maybe just reuse the `MergeTreesAction` (and only show it for different trees)?
Do you agree?

Reply:
Hmm, I see what you mean. By adding this option, it is very easy to create a cyclic tree and thus this option encourages this.

I removed the possibility to add an edge between two nodes of the same tree :+1: